text/0000-cosign.md

```diff
@@ -0,0 +1,100 @@
+# Meta
+[meta]: #meta
+- Name: Cosign support
+- Start Date: 2021-12-03
+- Author(s): [@samj1912](https://github.com/samj1912)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
```

Comment:
This RFC supports indeed to sign an existing SBoM but also to export the attestation or attachment. Summary text should be reviewed